.travis.yml

```diff
@@ -1,9 +1,17 @@
 language: node_js
+os:
+  - linux
+  - osx
+  - windows
+jobs:
+  allow_failures:
+    - os: windows
```

Comment:
why do we want to have this in CI but allow failures? this is not very useful

Reply:
I'd like to be able for folks like Willem to reproduce this and see it in action while we work to fix it.
Then at the end we just remove that one jobs key
Would you rather do it a different way?

Reply:
removed the windows check